src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -1149,3 +1149,8 @@ def handle_version_update():
             refresh_known_clouds()
     except Exception as ex:  # pylint: disable=broad-except
         logger.warning(ex)
+
+
+def start_powershell_process(ps_process_cmd):
```

Comment:
How about `powershell_start_process`, as it is not really a PowerShell process, but starting a process using PowerShell?

```suggestion
def powershell_start_process(ps_process_cmd):
```